Packs/fireeye/Playbooks/playbook-Detonate_URL_-_FireEye_AX.yml

```diff
@@ -279,29 +285,31 @@ tasks:
     continueonerrortype: ""
   '7':
     id: '7'
-    taskid: 922099fa-9796-48f4-8b19-090d46bda6ac
+    taskid: 32b33c57-a932-4709-826d-9c9f0e2d4118
     type: regular
     task:
-      id: 922099fa-9796-48f4-8b19-090d46bda6ac
+      id: 32b33c57-a932-4709-826d-9c9f0e2d4118
       version: -1
       name: Get task report
-      description: Results of the file submission.
-      script: fireeye|||fe-submit-result
```

Comment:
please revert this change - it should remain the same command  " fireeye|||fe-submit-result"